Ginger/Ginger/Actions/ActBrowserElementEditPage.xaml.cs

```diff
@@ -32,15 +34,21 @@ namespace Ginger.Actions
     public partial class ActBrowserElementEditPage : Page
     {
         private ActBrowserElement mAct;
-
+        PlatformInfoBase mPlatform;
         public ActBrowserElementEditPage(ActBrowserElement act)
         {
             InitializeComponent();
             mAct = act;
-           
+
+            ePlatformType activityPlatform = GetActivityPlatform();
+            mPlatform = PlatformInfoBase.GetPlatformImpl(activityPlatform);
+
```

Comment:
we can't calcualte the Platform type if the Action is on Shared Repository as stand alon so we need to serilize the platform after calcualting it for first time- see what i done in UIElement Action edit page